docs(KeepCache): deploy without cache clear (#121)
moovweb-docs/xdn-docs: guides/caching.md

![deployments](/images/caching/purge.png)

> ## Deploy without cache clear

Reviewer: ```suggestion
## Preserving the Cache on Deploy
```

> By default, XDN clears your environment edge cache on every new deploy.

Reviewer 2: ```suggestion
By default, XDN clears your environment edge cache on every time you deploy a new version of your site.
```

> This behaviour can be turned off in [Environment](environments) options by ticking the following checkbox:

Reviewer 2: ```suggestion
This behavior can be turned off in [Environment](environments) options by enabling the following option:
```
